spec/models/profile_spec.rb

```diff
@@ -176,8 +176,8 @@
       @profile[:image_url] = 'large'
       @profile[:image_url_small] = nil
       @profile[:image_url_medium] = nil
-      expect(@profile.image_url(:thumb_small)).to eq('large')
-      expect(@profile.image_url(:thumb_medium)).to eq('large')
+      expect(@profile.image_url(size: :thumb_small)).to eq('large')
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.